Zappy_IA_Report/NN/Code/Tutorial6.py

```
@@ -0,0 +1,67 @@
+
+def review_encode(s):
+	encoded = [1]
+	for word in s:
+		if word.lower() in word_index:
+			encoded.append(word_index[word.lower()])
+		else:
+			encoded.append(2)
+
+	return encoded
+
+with open("text.txt", encoding="utf-8") as f:
```

Flake8 (7.2.0)

[error] expected 2 blank lines after class or function definition, found 1

(E305)